ftdetect/pfw.vim

```diff
@@ -2,4 +2,4 @@
 au BufRead,BufNewFile *.{pfw}   set filetype=pfw"
 
 " Disable expandtab since pfw tabs are part of language
-autocmd FileType pfw set noet sw=4 sts=4 ts=4 tabstop=4
+autocmd FileType pfw set noet sw=4 sts=4 ts=4 tabstop=4 commentstring=#%s
```

Comment:
I guess the commentstring is used by plugins to get some context in order to comment out blocs, for instance?

You are adding a trailing whitespace at line 5.
Please remove it and resubmit a PR